src/NLog.Extensions.Logging/Logging/NLogLoggerFactory.cs

```diff
@@ -86,9 +84,7 @@ public Microsoft.Extensions.Logging.ILogger CreateLogger(string categoryName)
         /// <param name="provider">The <see cref="T:Microsoft.Extensions.Logging.ILoggerProvider" />.</param>
         public void AddProvider(ILoggerProvider provider)
         {
-            InternalLogger.Debug("AddProvider will be ignored");
+            InternalLogger.Debug("NLogLoggerFactory: AddProvider has been ignored {0}", provider?.GetType());
```

Comment:
maybe guard with InternalLogger.IsDebugEnabled?

Reply:
It only happens at application startup when doing wiring of the application.